dao/src/testFixtures/kotlin/Fixtures.kt

```diff
@@ -188,15 +189,24 @@ class Fixtures(private val db: Database) {
         return Jobs(analyzerJob, advisorJob, scannerJob, evaluatorJob, reporterJob, notifierJob)
     }
 
-    fun createIdentifier() = db.blockingQuery {
+    fun createIdentifier(identifier: Identifier): Identifier = db.blockingQuery {
```

Comment:
Instead of duplicating the function you could provide a default argument:

```kotlin
fun createIdentifier(identifier: Identifier = Identifier("identifier_type", ...)): Identifier = ...
```

Reply:
No problem. However it doesn't look cleaner than two methods.